enterprise/frontend/src/embedding-sdk/cli/constants/messages.ts

```diff
@@ -30,6 +32,11 @@ export const EMBEDDING_FAILED_MESSAGE = `
 export const PREMIUM_TOKEN_REQUIRED_MESSAGE =
   "  Don't forget to add your premium token to your Metabase instance in the admin settings! The embedding demo will not work without a license.";
 
+export const NO_TENANCY_COLUMN_WARNING_MESSAGE = `
+  Your have not selected any tables with a multi-tenancy column.
+  You can still use the SDK, but you will not be able to sandbox your tables.
+`;
```

Comment:
This happens when you've chosen to use multi-tenancy, but you have not picked a tenancy column for any of your tables. This warns the user that the multi-tenancy configuration will be abandoned.